spec/model/postgres/postgres_server_spec.rb

```diff
@@ -310,10 +341,10 @@ def create_standby_resource(suffix)
 
       resource.update(parent: parent_resource)
       postgres_server.update(timeline_access: "fetch")
-      allow(resource.parent.representative_server).to receive(:current_lsn).and_return("F/F")
     end
 
     it "returns true if the diff is less than 80MB" do
```

Comment:
FWIW, this spec seems weirdly named. `F/F` doesn't indicate "less than 80MB" to me. Maybe it indicates the difference is `0`, and that is a useful test case. However, maybe we should add a spec that uses a numeric example less than 80MB?